app/models/user.rb

```diff
@@ -23,6 +23,7 @@ class User
   field :permission_groups, type: Array, default: ['user']
 
   field :password_digest
+  field :subscribe_newsletter, type: Boolean, default: false
```

Comment:
One thing I notice is that we'll be saving the value of `subscribe_newsletter`, but we don't have any mechanism for keeping it in sync. I'm not sure that we'll need this field again after creation, so it seems like we could  use `attr_accessor` instead of `field` and then do the check in the `:after_create` callback. What are your thoughts on that approach?